packages/gatsby/src/internal-plugins/internal-data-bridge/gatsby-node.js

```diff
@@ -157,3 +157,7 @@ emitter.on(`DELETE_PAGE`, action => {
   const node = getNode(nodeId)
   boundActionCreators.deleteNode({ node })
 })
+
+emitter.on(`BOOTSTRAP_FINISHED`, () =>
+  boundActionCreators.setProgramStatus(`BOOTSTRAP_FINISHED`)
```

Comment:
Should we move this [here](https://github.com/gatsbyjs/gatsby/blob/0df55f61587fe7f86e48bba5599d3fdc5d59b19b/packages/gatsby/src/bootstrap/index.js#L523)?

Reply:
Yeah, that'd make more sense 👍